### DISCUSSION TITLE
Laden startet automatisch beim Einstecken des Kabels

### DISCUSSION THREAD

**Discussion by herbert131** [goecharger/go-eCharger-API-v2#195] [Q&A]
hallo,
wie verhindere ich, dass das Laden automatisch beim Einstecken des Kabels startet. Bisher habe acs=1 und trx=1 gesetzt und dachte hiermit steuere ich dies. Hat sich das Verhalten mit der neuen SW (55.8) verändert ?

**Comment by 0xFEEDC0DE64** [comment]
acs=1 macht, dass Authentifizierung erforderlich ist.
trx=1 macht, dass eine Authentifizierung vorliegt.

Wenn nicht geladen werden soll, dann einfach nicht trx=1 senden zb.
oder trx=null beendet eine Transaction vorzeitig.

**Reply by 0xFEEDC0DE64** [reply]
oder verstehe ich es falsch und er lädt bereits bevor trx=1 gesendet wird mit acs=1? weil dann bräuchte ich bitte eine seriennummer und wir schauen mal nach, warum er geladen hat

**Reply by herbert131** [reply]
ich versuchs kurz darzulegen (acs = 1 ist gesetzt):
ich setze trx = 0 um zu verhindert das geladen wird
wenn ich das Ladekabel anschließe bleibt trx = 0 (modelstatus = 15) für ca 30 Sekunden
dann wechselt trx auf "null" ((modelstatus = 2)
Ladestart : setzt trx=1 -> und starte das laden
Nach ladestopp (frc) versuche ich wieder trx = 0 zu setzen was misslingt: failed: cannot start another transaction when it has already been used to charge (0)  
vielleicht mach ich es falsch oder zu kompliziert ;-) ?

**Reply by herbert131** [reply]
als Ergänzung trx lässt sich nach dem Beenden des Ladens - bei weiter angestecktem Ladekabel weder auf null noch 0 setzen sondern beleibt auf 1 wenn es vorher gesetzt wurde

**Reply by 0xFEEDC0DE64** [reply]
trx=0 startet die transaction so, als hätte es die app gestartet
trx=1 startet die transaction so, als hätte die erste rfid karte sie gestartet
trx=10 startet die transaction so, als hätte die zehnte rfid karte sie gestartet
usw....
trx=null beendet die transaction dann wieder (muss man nicht setzen wenn das auto frisch angesteckt wurde)

**Comment by martiko70** [comment]
Hi,
ich hänge mich hier mal dran, weil zumindest der Anfang (der ja funktioniert) das ist, was ich gerne auch umsetzen möchte. Konkret würde ich das Laden (über eine Logik in node Red) starten, wenn mein Balkonkraftwerk mehr Strom erzeugt, als ich gerade verbrauche.
Aber... ich habe RFID-Authentifizierung aktiviert und möchte den Verbrauch dann auch einer bestimmten RFID-Kennung zuordnen. Geht das irgendwie? Oder anders gefragt, welcher RFID-Kennung wird denn der Ladevorgang zugeordnet, wenn man es per trx=1 startet (ohne vorher wirklich einen RFID-Chip dran gehalten zu haben).
